util/src/main/java/org/killbill/billing/util/config/definition/InvoiceConfig.java

```diff
@@ -72,6 +72,16 @@
     @Description("Whether we disable writing $0 usage amounts")
     boolean isUsageZeroAmountDisabled(@Param("dummy") final InternalTenantContext tenantContext);
 
+    @Config("org.killbill.invoice.usage.missing.lenient")
```

Comment:
@reshmabidikar Juste a note to remember to update our config docs with this new property.